common/scala/src/main/scala/whisk/core/connector/Message.scala

```diff
@@ -53,7 +54,8 @@ case class ActivationMessage(
     activationId: ActivationId,
     activationNamespace: EntityPath,
     content: Option[JsObject],
-    cause: Option[ActivationId] = None)
+    cause: Option[ActivationId] = None,
+    traceMetadata: Option[SpanMetadata] = None)
```

Comment:
> is this something which can be done independently on similar lines as you and @ddragosd had discussed in previous comments?

no need to wait (as the log drain integration hasn't gotten to the point of building a load time solution yet); OSGI or URLClassLoader with the latter being more lightweight to start.